Examples/UtiliKit-iOSExample/Collection/Cells, Supplementaries and Decorations/Cells/NibDrivenCell.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  NibDrivenCell.swift
+//  UtiliKit
+//
+//  Copyright © 2021 Bottle Rocket Studios. All rights reserved.
+//
+
+import UIKit
+import UtiliKit
+
+class NibDrivenCell: UICollectionViewCell { }
+
+// MARK: - Configurable
+extension NibDrivenCell: Configurable {
+
+    func configure(with color: UIColor) {
```

Comment:
Personal preference, but since "configure" is often used in terms of configuring a cell with its data model, I might suggest we name this something like "configure background color" to make it a bit more clear.
```suggestion
    func configureBackgroundColor(_ color: UIColor) {
```

Reply:
I can't rename in this instance because I'm using conformance to `Configurable` so that I can demonstrate usage of `func dequeueReusableCell<T: UICollectionViewCell & Configurable>(for indexPath: IndexPath, configuredWith element: T.ConfiguringType) -> T`.

I'm happy to change the method name, but then we'll have to lose the demonstration of that part of the UtiliKit functionality. What do you think?

Reply:
I thought I looked for the `Configurable` conformance, but I must be blind. It's fine to keep it as-is to demonstrate that functionality.

Reply:
I went ahead and changed the associated type from `UIColor` to a `Configuration` struct that contains a `backgroundColor` property to make it more explicit here (same with the supplementary views and their titles)